src/main/flight/mixer.c

```diff
@@ -528,36 +556,13 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs)
         throttle = gpsRescueGetThrottle();
     }
 #endif
-
-#ifdef USE_AIRMODE_LPF
```

Comment:
The AIRMODE_LPF code seems to have gone missing - this should probably go inside `applyMixerAdjustment()`.

Reply:
Yes that is intentional. AIRMODE_LPF should go and needs further cleanup. I suggest to clean it up in other PR?

Reply:
AIRMODE_LPF is off by default, so there is no harm in leaving it in. What is the reason for wanting to remove it?

And if we want to remove it we should be removing it completely, and in a separate pull request.

Reply:
I agree with a separate PR to remove AIRMODE_LPF, which should be actioned before this PR.

Reply:
I removed it in another commit in this PR anyway

Reply:
It is added back now